src/main/java/org/recordrobotics/charger/commands/manual/ManualClaw.java

```diff
@@ -0,0 +1,60 @@
+package org.recordrobotics.charger.commands.manual;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import org.recordrobotics.charger.subsystems.Claw;
+import org.recordrobotics.charger.control.IControlInput;
+
+
+public class ManualClaw extends CommandBase {
+
+	private Claw _claw;
+	private IControlInput _controls;
+
+	private static final double TURN_SPEED = 0.3;
+
+	public ManualClaw(Claw claw, IControlInput controls) {
+		if (claw == null) {
+			throw new IllegalArgumentException("claw is null");
+		}
+		if (controls == null) {
+			throw new IllegalArgumentException("Controls is null");
+		}
+
+		_claw = claw;
+		_controls = controls;
+		addRequirements(claw);
+	}
+
+	@Override
+	public void execute() {
+		switch (_controls.getClawTurn()) {
+			case CUBE:
+				if(Claw.CLAW_CUBE - _claw.getPosition() > 0) {	
+					_claw.turn(TURN_SPEED);
+				} else if(Claw.CLAW_CUBE - _claw.getPosition() < 0) {
+					_claw.turn(-TURN_SPEED);
+				} else {
+					_claw.turn(0);
+				}
+				return;
+			case NEUTRAL:
+				_claw.turn(-TURN_SPEED);
```

Comment:
Neutral should not turn indefinitely. It should be based on position as well.

Reply:
Neutral is a state where there is no Driver input, it will stop once it returns to the open state, then it will set speed to 0.

Reply:
turn should also automatically stop the claw at neutral

Reply:
Okay, I see the logic now. It's a little strange that it's split up between different files.
I would recommend a bit of refactoring
- Put all the logic in one place, so it's easy to read
- Combined repeated code segments, like the if/else's in this function (notice only one value is different)

Reply:
should the original code still exist as a backup? (so the claw doesn't hit the arm)

Reply:
There should be no need.
The logic will remain the same ideally.